transform/mattermost-analytics/models/marts/marketing/_marketing__models.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+
+models:
+  - name: fct_security_newsletter
+    description: |
+      Security Newsletter data from cws marketing, to sync with leads & campaign member in Salesforce.
+
+    columns:
+      - name: marketing_id
+        description: Unique identifier for cws marketing data.
+      - name: email
+        tags: ['pii']
+        description: Customer email for marketing coming from cws.
+        tests:
+          - not_null
+          - dbt_utils.not_empty_string
+      - name: server_id
+        description: Server ID of the user who subscribed to the marketing campaign.
+      - name: lead_id
+        description: Lead ID of the user who subscribed to the marketing campaign in Salesforce.
```

Comment:
It would be nice to add the details about how the lead id is selected in case of multiple lead ids.

Reply:
Added more detail about the logic.